temple.rb

```diff
@@ -0,0 +1,17 @@
+class Temple < Formula
+  desc "A Software Framework for automatically generating microservices from a high level DSL"
+  version "0.001"
+  homepage "https://templeeight.github.io/"
+  url "https://github.com/TempleEight/temple/releases/latest/download/temple-latest"
```

Comment:
This shouldn’t be `latest`

Reply:
Do you think we should put a PR in for every release?

Reply:
This has a version and a checksum. We definitely need to